StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift

```diff
@@ -224,17 +224,43 @@ extension TextFieldElement {
     }
 }
 
+// MARK: - Censored CVC Configuration
+extension TextFieldElement {
+    struct CensoredCVCConfiguration: TextFieldElementConfiguration {
+        init(cardBrandProvider: @escaping () -> (STPCardBrand)) {
+            self.cardBrandProvider = cardBrandProvider
+            let maxLength = Int(STPCardValidator.maxCVCLength(for: cardBrandProvider()))
+            self.defaultValue = String(repeating: "•", count: maxLength)
+        }
+
+        let defaultValue: String?
+        let cardBrandProvider: () -> (STPCardBrand)
```

Comment:
Is there any reason to pass in the provider rather than just the brand? Seems like we can just pass in the brand.